src/System.Private.Uri/src/System/UriFormatException.cs

```diff
@@ -2,12 +2,22 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#if netstandard10
```

Comment:
All these #ifdef's kind of make me sad.  What build configuration are they being included in? I don't see a netstandard1.7 build configuration. 

At any case we should try to eliminate the need for all these just because of serialization. We should do what @ericstj has been suggesting of having a S.R.Serialization.Formatters library that targets an older netstandard that has these dummy attributes and interfaces (or we should just include local versions of that source from common in the older build configuration).